src/Behat/Behat/Context/Reader/AnnotatedContextReader.php

```diff
@@ -132,6 +133,18 @@ private function readDocBlockCallees($class, ReflectionMethod $method, $docBlock
     }
 
     /**
+     * Merges multiline strings (strings ending with "\")
+     *
+     * @param string $docBlock
+     *
+     * @return string
+     */
+    private function mergeMultilines($docBlock)
+    {
+        return preg_replace("#\\\\$\s*\*\s*([^\\\\$]+)#m", '$1', $docBlock);
```

Comment:
btw, backtracking is even worse on PHP 7: some kind of backtracking can make PCRE reach the JIT stack limit much sooner than reaching the backtrack limit (set to 1000000 in PHP), so regexes working (inefficiently) on PHP 5.6 could break on PHP 7 for the same input